### ISSUE TITLE
Error on import

### ISSUE THREAD

**Issue by peni4142** [philipphecht/react-native-doc-viewer#86]
I installed it like the docs:

`npm install react-native-doc-viewer --save`

` react-native link react-native-doc-viewer`

but when I import it to my Project, I get an Error:
`import OpenFile from 'react-native-doc-viewer';`
Error: undefined is not an object (evaluating 'RNReactNativeDocViewer.openDoc')

react-native info:
Environment:
  OS: Windows 10
  Node: 6.11.3
  Yarn: 1.5.1
  npm: 5.5.0
  Watchman: Not Found
  Xcode: N/A
  Android Studio: Version  3.1.0.0 AI-173.4670197

Packages: (wanted => installed)
  react: 16.3.0-alpha.1 => 16.3.0-alpha.1
  react-native: 0.52.0 => 0.52.0

**Comment by peni4142** [comment]
now everything is up to date
package.json:
{
  "name": "ProdPrototype",
  "version": "0.1.0",
  "private": true,
  "devDependencies": {
    "jest-expo": "26.0.0",
    "react-native-scripts": "1.13.1",
    "react-test-renderer": "16.3.0-alpha.1"
  },
  "main": "./node_modules/react-native-scripts/build/bin/crna-entry.js",
  "scripts": {
    "start": "react-native-scripts start",
    "eject": "react-native-scripts eject",
    "android": "react-native-scripts android",
    "ios": "react-native-scripts ios",
    "test": "jest"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "expo": "^26.0.0",
    "react": "16.3.0-alpha.1",
    "react-native": "0.54.0",
    "react-native-doc-viewer": "^2.7.8",
    "react-navigation": "^1.5.9"
  }
}

**Comment by mkjfeng01** [comment]
@peni4142 Same problem
`"react": "^16.3.0-rc.0",
"react-native": "^0.55.0",
"react-native-doc-viewer": "^2.7.8"`

![](https://ws4.sinaimg.cn/large/006tKfTcgy1fq0rk0o0z5j31kw0qatcb.jpg)

**Comment by peni4142** [comment]
I have no Xcode...

**Comment by mkjfeng01** [comment]
@sam1463 nice!

**Comment by sam1463** [comment]
@peni4142 good point, my comment only fixes on ios. Not sure how to manually import on windows...

**Comment by mkjfeng01** [comment]
@sam1463 maybe he said Android...

**Comment by tsurumure** [comment]
I have this problem, Can it be solved?

**Comment by anil1712** [comment]
I am getting the same issue. I want it for Android only. 

FYI: I am using expo with React Native

**Comment by sam1463** [comment]
For those of you having issues with the automatic installation on android, have you also tried using the [manual android installation instructions](https://github.com/philipphecht/react-native-doc-viewer#android)?

**Comment by thinklinux** [comment]
I installed it again and run the link command and everything worked out. Have no idea what happened :D

**Comment by zypher2004** [comment]
I had this happen when I forgot to run `pod install` after react-native link
